internal/app/connectconformance/testsuites/data/connect_client_error_endstream.yaml

```diff
@@ -107,6 +107,73 @@ testCases:
       error:
         code: CODE_OUT_OF_RANGE
         message: oops
+  - request:
+      # Clients must not depend on data in the "debug" key when deserializing details.
+      testName: error/disregard-detail-debug
```

Comment:
I think the test cases are typically named in a way that describe the scenario, vs describing the implementation's expected behavior. How about `invalid-details-debug` instead of `disregard-detail-debug`? (Sorry I didn't mention this with previous remark.)

Reply:
👍  There are a couple of existing tests named "allow-unrecognized...". Doesn't seem worth renaming them, but renamed the new ones in https://github.com/connectrpc/conformance/pull/921/commits/7a9e8c2f105f89a11a8c84c04842f2f34ae32662.